project/app/src/main/java/com/nexte/nexte/LoginScene/LoginView.kt

```diff
@@ -1,37 +1,106 @@
 package com.nexte.nexte.LoginScene
 
+import android.content.Intent
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
-import android.widget.Button
-import android.widget.EditText
-import android.widget.TextView
+import android.view.View
+import com.facebook.accountkit.AccountKit
+import com.facebook.accountkit.AccountKitLoginResult
+import com.facebook.accountkit.ui.AccountKitConfiguration
+import com.facebook.accountkit.ui.AccountKitActivity
+import com.facebook.accountkit.ui.LoginType
+import com.nexte.nexte.R
 import kotlinx.android.synthetic.main.activity_login_view.*
-import kotlinx.android.synthetic.main.activity_login_view.view.*
-import kotlinx.android.synthetic.main.activity_main.*
 
 interface LoginDisplayLogic {
 
     fun displayAuthenticateState(viewModel: LoginModel.ViewModel)
 }
 
-/*
-Commented cause it should make reference to main view at this point
+
 
 class LoginView : AppCompatActivity(), LoginDisplayLogic {
 
     var interactor: LoginBusinessLogic? = null
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login_view)
+        this.setup()
+
+        AccountKit.initialize(this) // Account Kit
+
+        requestAuthentication.setOnClickListener {
 
-            val request: LoginModel.Request = LoginModel.Request(usernametext.toString(),
-                                                                 passwordtext.toString())
+            val account = accountField.text.toString()
+            val password = passwordField.text.toString()
+
+            val request: LoginModel.Request = LoginModel.Request(account, password)
             this.interactor?.doAuthentication(request)
+        }
+
     }
 
-    override fun displayAuthenticateState(viewModel: LoginModel.ViewModel) {
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+
 
+        if (requestCode == LoginModel.AuthenticationStatus.FACEBOOK_REQUEST.value) {
+
+            val loginResult = data!!.getParcelableExtra<AccountKitLoginResult>(AccountKitLoginResult.RESULT_KEY)
+            var message = ""
+
+            if (loginResult.error != null) {
+                println("1 $loginResult?.error")
+            } else if (loginResult.wasCancelled()) {
+                message = "Login cancelled"
+                println("2 $message")
+            } else {
+                if (loginResult.accessToken != null) {
+                    message = "Sucess" + loginResult.accessToken
+                    println("1 $message")
+                } else {
+                    println(loginResult.authorizationCode)
+                }
+            }
+        }
     }
-}
 
-*/
\ No newline at end of file
+    override fun displayAuthenticateState(viewModel: LoginModel.ViewModel) { }
```

Comment:
??? Cadê a conclusão desse método?

Reply:
Isso vai ficar assim, será um popup, ou mensagem que será provida pelo design, tudo que for feito nisso terá que ser refatorado.